scout/supporting_data/config_schema.yml

```diff
@@ -0,0 +1,243 @@
+type: object
+required: [description]
+properties:
+  description:
+    type: ["string", "null"]
+    description: A description of the scenario configuration.
+  ecm_prep:
+    type: object
+    required: []
+    properties:
+      site_energy:
+        type: boolean
+        default: false
+        description: Enable site energy calculation.
```

Comment:
For booleans, how is the description phrased relative to the default? If the default is `false` and the description starts with "Enable" does that mean that setting to `true` "disables" the option in question?

Reply:
I updated all bools to start with "If true, ..." to improve clarity